src/send.c

```diff
@@ -376,7 +381,7 @@ int send_run(sock_t st, shard_t *s)
 			zconf.probe_module->make_packet(
 			    buf, &length, src_ip, current_ip,
 			    htons(current_port), ttl, validation, i,
-			    probe_data);
+			    aesrand_getword(aes_rand_gen), probe_data);
```

Comment:
I think that this will be the same for every packet that goes out, if I'm reading this code correctly, not random for each packet, which is what you want. I think that you can just use the last 4 bytes of validation here since probe modules generally only use the first few bytes.

Reply:
Actually I take that back, I just looked at the code, and it does look like it works as you expected. My apologies.